### PR TITLE
docs(v2): add instructions on Travis CI deployment

### PR DESCRIPTION
**Pull request by mohamedsgap**
I created this PR to update Docusaurus V2 Doc/deployment with Automating Deployments steps with Travis CI.
Ref Issue: #2286

#### Motivation
Because When I tried to use Travis CI with Docusaurus V2, I didn't find any steps for automating deployments using CI with Travis CI or even Circle CI, so I jumped back to Docusaurus V1 and I made a some update to Travis CI script to be suitable with Docusaurus V2.

### CODE REVIEW

File: website/versioned_docs/version-2.0.0-alpha.41/deployment.md [hunk 1]
```diff
@@ -146,3 +146,35 @@ Render offers [free static site hosting](https://render.com/docs/static-sites) w
    | **Publish Directory** | `build`       |
 
 That's it! Your app will be live on your Render URL as soon as the build finishes.
+
```
Comment (yangshun) [inline website/versioned_docs/version-2.0.0-alpha.41/deployment.md]: Please remove this change. We should just update the newest docs.

File: website/docs/deployment.md [hunk 2]
```diff
@@ -146,3 +146,35 @@ Render offers [free static site hosting](https://render.com/docs/static-sites) w
...
+1.  Using your GitHub account, [add the Travis CI app](https://github.com/marketplace/travis-ci) to the repository you want to activate.
+1.  Open your Travis CI dashboard. The URL looks like https://travis-ci.com/USERNAME/REPO, and navigate to the `More options` > `Setting` > `Environment Variables` section of your repository.
+1.  Create a new environment variable named `GH_TOKEN` with your newly generated token as its value, then `GH_EMAIL` (your email address) and `GH_NAME` (your GitHub username).
+1.  Create a `.travis.yml` on the root of your repository with below text.
+
+```yaml
+# .travis.yml
+language: node_js
+node_js:
+  - '8'
```
Comment (yangshun) [inline website/docs/deployment.md]: We support Node 10 and above.

**Comment by mohamedsgap** [comment]
Hey @yangshun I've just made the suggested changes :)

File: website/versioned_docs/version-2.0.0-alpha.43/deployment.md [hunk 3]
```diff
@@ -146,3 +146,35 @@ Render offers [free static site hosting](https://render.com/docs/static-sites) w
    | **Publish Directory** | `build`       |
 
 That's it! Your app will be live on your Render URL as soon as the build finishes.
+
+## Automating Deployments Using Continuous Integration
```
Comment (lex111) [inline website/versioned_docs/version-2.0.0-alpha.43/deployment.md]: @mohamedsgap please remove change in this file. Docs for version 43 has already been released, so we cannot and should not add anything new to this version.

**Comment by mohamedsgap** [comment]
Hi @lex111, just removed the update from v2.0.0-alpha.43

**Pull request merged**